<launch>
  <node pkg="jsk_pcl_ros" type="supervoxel_segmentation"
	name="object_supervoxel" output="screen">
    <remap from="~input" to="/object_model/output/cloud" />
  </node>
  <node pkg="jsk_pcl_ros" type="cluster_point_indices_decomposer"
	name="object_supervoxel_decomposer">
    <remap from="~input" to="object_supervoxel/output/cloud" />
    <remap from="~target" to="object_supervoxel/output/indices" />
  </node>
</launch>
